frontend/src/pages/ActivityReport/Pages/nextSteps.js

```diff
@@ -1,21 +1,193 @@
+          humanName={humanName}
+        />
+      </Fieldset>
+    );
+  }
+
+  const targetId = notes[targetIndex] ? notes[targetIndex].id : undefined;
+  const defaultValue = notes[targetIndex] ? notes[targetIndex].note : undefined;
+  return (
+    <>
+      <Fieldset className="smart-hub--report-legend smart-hub--form-section" legend={`${humanName} Next Steps`}>
+        <ul className="usa-list--unstyled">
+          {notes.map((item, index) => (
+            <li key={item.note} className="grid-row flex-row border-bottom padding-top-2 padding-bottom-2" style={{ borderColor: '#f0f0f0' }}>
+              <div className="grid-col flex-12">
+                {item.note}
+              </div>
+
+              <div className="grid-col" style={{ marginTop: '0px' }}>
+                <ContextMenu
+                  label="Menu"
+                  menuItems={
+                    [
+                      { label: 'Edit', onClick: (e) => { e.preventDefault(); onEdit(index); } },
+                      { label: 'Delete', onClick: (e) => { e.preventDefault(); onDelete(index); } },
+                    ]
+                  }
+                />
+              </div>
+            </li>
+          ))}
+        </ul>
+
+        {showPrompt ? (
+          <div className="border-left-05 border-blue padding-left-2 smart-hub-border-blue-primary">
+            <NoteEntry
+              onEntry={(value) => onEntry(value, targetIndex, targetId)}
+              isRequired={false}
+              onCancel={onCancel}
+              name={name}
+              humanName={humanName}
+              defaultValue={defaultValue}
+            />
+          </div>
+        )
+          : (
+            <Button unstyled onClick={() => onEdit(notes.length)}>
+              <FontAwesomeIcon icon={faPlusCircle} />
+              <span className="padding-left-05">Add New Follow Up</span>
+            </Button>
+          )}
+
+      </Fieldset>
+    </>
+  );
+};
+
+NoteEntries.propTypes = {
+  name: PropTypes.string.isRequired,
+  humanName: PropTypes.string.isRequired,
+};
+
 const NextSteps = () => (
   <>
     <Helmet>
       <title>Next steps</title>
     </Helmet>
-    <div>
-      Next Steps
+
+    <div className="padding-bottom-205">
+      <NoteEntries name="specialistNextSteps" humanName="Specialist" />
     </div>
+
+    <NoteEntries name="granteeNextSteps" humanName="Grantees" />
+
   </>
 );
 
-NextSteps.propTypes = {};
-
-const sections = [];
+const sections = [
+  {
+    title: 'Specialist next steps',
+    anchor: 'specialist-next-steps',
+    items: [
+      { label: 'What have you agreed to do next?', name: 'specialistNextSteps', path: 'note' },
+    ],
+  },
+  {
+    title: 'Grantee next steps',
+    anchor: 'grantee-next-steps',
+    items: [
+      { label: 'What have you agreed to do next?', name: 'granteeNextSteps', path: 'note' },
+    ],
+  },
+];
 
 const ReviewSection = () => (
   <ReviewPage sections={sections} path="topics-resources" />
```

Comment:
Can you update this path? It is a copy/paste error that I made. It should be `next-steps`